packages/conf-autoconf.0.1/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "build": [ [ "which", "autoconf"] ],
```

Comment:
Let's set it to 'true' instead

Reply:
U mean `"build": true`? what does it mean?